### PR TITLE
Add More Color Breaks

### PR DESCRIPTION
**Pull request by jamesmcclain**
# First Commit

Breaks for NVDI, VARI, and VGI have been added.

Connects #35
## Testing Instructions
- Make sure that the code builds
- Run the jar, preferably on your host so that you can see the output from it on the console; Confirm that NVDI, VGI, and VARI are loaded.
- Compare the NVDI, VGI, and VARI files with the breaks given in azavea/modellab#151 to make sure that they were copied correctly.
# Second Commit

A "coerce" operation has been added to allow floating-point rasters to result from integral raster inputs.

Connects #37
## Testing Instructions

Choose one of the following:
### Choice 1

Post json blob to the server:

```
{
    "name": "LocalDivide",
    "breaks_name": "ndvi",
    "inputs": [{
        "name": "LocalSubtract",
        "inputs": [{
            "name": "LoadLayer",
            "layer_name": "us-pennsylvania-philadelphia-landsat-nir-30m-epsg3857",
            "layer_alias": "Landsat NIR",
            "hash": "COZ373Y"
        }, {
            "name": "LoadLayer",
            "layer_name": "us-pennsylvania-philadelphia-landsat-red-30m-epsg3857",
            "layer_alias": "Landsat Red",
            "hash": "3BWIPNI"
        }]
    }, {
        "name": "LocalAdd",
        "inputs": [{
            "name": "Coerce",
            "inputs": [{
                "name": "LoadLayer",
                "layer_name": "us-pennsylvania-philadelphia-landsat-nir-30m-epsg3857",
                "layer_alias": "Landsat NIR",
                "hash": "COZ373Y"
            }]
        }, {
            "name": "LoadLayer",
            "layer_name": "us-pennsylvania-philadelphia-landsat-red-30m-epsg3857",
            "layer_alias": "Landsat Red",
            "hash": "3BWIPNI"
        }]
    }]
}
```

Visit [this endpoint](http://localhost:8888/sample/MSSM45Q/12/-75.13893127441406/39.97554154865437?bufferSize=2) and verify that there are floating-point results.  (The assumption is that you started the code using `spark-submit` on your host.)
### Choice 2

Test along with the solution for azavea/modellab#191.
# Third Commit

The operation name in echoed structure input would be e.g. "LocalLocalUnarySqrt" prior to this change, now it is just "LocalUnarySqrt".

Connects #34

### CODE REVIEW

File: src/main/scala/com/azavea/modellab/Service.scala [hunk 1]
```diff
@@ -68,30 +72,17 @@ object StaticConfig {
     "landsat",
     "nlcd",
     "nlcd-reclass",
+    "nvdi",
```
Comment (rajadain) [inline src/main/scala/com/azavea/modellab/Service.scala]: This should be `ndvi`, not `nvdi`

Comment (jamesmcclain) [inline src/main/scala/com/azavea/modellab/Service.scala]: Thanks

**Comment by jamesmcclain** [comment]
All comments prior to this one have been addressed.

**Comment by rajadain** [comment]
TRI renders the best. NDVI and VARI both suffer from a cyan-out due to 0 values being returned from the back-end, seen here:

image

VGI shows nothing, likely because it is multiplying the 0 output of NDVI with 150:

image

But I don't think that these issues are because of the work done here, which seems pretty straightforward. Consider looking in to these issues as part of this card, but if they are found to be significantly complex than create and point separate cards for them.

**Comment by jamesmcclain** [comment]
I am going to add in the floating-point raster support before asking to merge this

**Comment by jamesmcclain** [comment]
I have added support for coercing to floats.

**Comment by rajadain** [comment]
Tested with https://github.com/azavea/modellab/pull/201

+1!

All tools render correctly now:

image

**Comment by jamesmcclain** [comment]
Okay, thanks for the review.  I will make the suggested changes on Friday.

**Pull request merged**